apps/web/app/(app)/environments/[environmentId]/settings/components/SettingsNavbar.tsx

```diff
@@ -192,7 +192,7 @@ export default function SettingsNavbar({
 
   return (
     <>
-      <div className="fixed hidden h-full bg-white py-2 pl-4 pr-10 md:block ">
+      <div className="fixed hidden h-full overflow-y-scroll bg-white py-2 pl-4 pr-10 md:block ">
```

Comment:
Hi thank you for your contribution 🎉 , tried testing this on a 13" Mac but still the issue isn't resolved. Even on 100% zoom level not able to view the `License` link in the Compliance section. Attaching both 100% and 90% views. Can you please fix this, will re-review it once you are done with the fix.

  
<img width="1438" alt="Screenshot 2023-10-23 at 5 59 00 PM" src="https://github.com/formbricks/formbricks/assets/31568159/c8151f9e-0148-450b-99fd-5def507d3af7">


<img width="1440" alt="Screenshot 2023-10-23 at 5 58 46 PM" src="https://github.com/formbricks/formbricks/assets/31568159/82ba1ebc-cc4d-452e-aa74-317aa9802ad6">


cc: @mattinannt

Reply:
@neilchauhan2 yes, you are right. Unfortunately the contributor hasn't answered and I would like to merge this issue to complete the hackathon. Can you please approve the PR? :-)